### ISSUE TITLE
Allow changing an alerts state after its ack'd if the message that would have been sent has changed

### ISSUE THREAD

**Issue by atammy-narmi** [opensearch-project/OpenSearch#8258]
**Is your feature request related to a problem? Please describe.**
Say I have multiple servers which are being monitored using an alert by looking at metricbeat stats (e.g. high CPU usage/ disk usage) and I get an alert notification saying 2 of the servers have high CPU usage and I ack'd the alert. Now if a third server also starts having high CPU usage I am not going to get a notification for this as the alert has already been ack'd.

**Describe the solution you'd like**
Allow resetting the alerts state/ or resending a notification even after an alert is acknowledged but if the message that would have been sent has changed.

**Describe alternatives you've considered**
I would have to create a separate alert for each server which is not ideal.
 
**Additional context**
N/A

**Comment by andrross** [comment]
@atammy-narmi  Are you using the [alerting plugin](https://github.com/opensearch-project/alerting) for this feature? If so, this issue will need to be transferred to that repo.

**Comment by atammy-narmi** [comment]
Yes, I am using the alerting plugin. Could you please transfer the issue there? Or should I close and open a new one there?

**Comment by Xtansia** [comment]
@opensearch-project/admin Can you please transfer this issue to https://github.com/opensearch-project/alerting
